add storage to filelog receiver for extraFileLog

omrozowicz-splunk opened the pull request:
This PR adds a `storage` field for `extraFileLog` if it wasn't provided in the `values.yaml`.
Fix for: https://github.com/signalfx/splunk-otel-collector-chart/issues/692

atoulme reviewed (approved):
Changes look good, example has the file_storage storage field set.

dmitryax commented on helm-charts/splunk-otel-collector/templates/config/_otel-agent.tpl:
```diff
@@ -405,7 +405,14 @@ receivers:
+  {{- range $extraFileLogKey, $extraFileLogValue := $extraFileLogsList }}
+  {{- printf "%s:" $extraFileLogKey | nindent 2 }}
+  {{- if not $extraFileLogValue.storage }}
+    storage: file_storage
```
I'm curios how this works with 2 spaces indent. I believe it should be 4, no?

omrozowicz-splunk replied:
Actually there are 4 spaces before `storage: file_storage`. I matched the indentation of the other elements of the filelog receiver.

dmitryax reviewed (commented):
@omrozowicz-splunk Please rebase and run `make render`

omrozowicz-splunk commented:
Added another commit to fix example file, as I thought it's such a small change it doesn't require testing (and forgot using mounts 🤯 ), now I simplified it and tested on real env. I deleted `another storage` case as well as it requires a ton of additional config and I'm not sure if it's even useful for anyone.
